pages.pl/common/cola.md

```diff
@@ -1,6 +1,6 @@
 # cola
 
-> Te polecenie jest aliasem `git-cola`.
+> To poleceniejest aliasem `git-cola`.
```

Comment:
Missing space. Should be "polecenie jest".

Reply:
Fixed, sorry :)